### PR TITLE
Enable macOS for CocoaPods release

### LINKED ISSUE #1514: Support macOS in Cocoapods releases
Can't install WalletCore with CocoaPods for macOS project.

It would be nice if macOS version was released in CocoaPods together with iOS version.

https://github.com/trustwallet/wallet-core/blob/master/tools/ios-release

### PR DESCRIPTION
**Pull request by grachyov**
## Description

Fixes #1514 

## Checklist

- [ ] Prefix PR title with `[WIP]` if necessary.
- [ ] Add tests to cover changes as needed.
- [ ] Update documentation as needed.
- [ ] I have read the [guidelines](https://developer.trustwallet.com/wallet-core/newblockchain#integration-criteria) for adding a new blockchain
- [ ] If there is a related Issue, mention it in the description (e.g. Fixes #<issue_number> ).

### CODE REVIEW

**Comment by hewigovens** [comment]
we disabled mac on purpose because after enabling bitcode, the final binary size bloated over 1G

**Comment by grachyov** [comment]
@hewigovens Would it be ok to make the macOS version a separate pod?

**Comment by hewigovens** [comment]
That's possible, we will revisit enabling mac later

File: WalletCore.podspec [hunk 1]
```diff
@@ -1,10 +1,10 @@
 version = '2.6.5'
 
 Pod::Spec.new do |s|
-  s.name         = 'WalletCore'
```
Comment (hewigovens) [inline WalletCore.podspec:4]: no need change this file, this is dev only podspec

**Comment by hewigovens** [comment]
@grachyov we tried Encrypted Ink, it looks still under development, but a cool idea to support

**Comment by grachyov** [comment]
@hewigovens thanks! This branch is no good anymore. I have used it to build macOS pod referencing forked repo.

I'll close this pull-request and submit a new one later.

Would it be better to release macOS version as a subspec or as a separate pod?

**Comment by vikmeup** [comment]
Please include in current spec, no need to create a new one. Thanks

**Pull request closed without merging**